sdk/bazel-java-deps.bzl

```diff
@@ -45,15 +45,17 @@ version_specific = {
 # This effectively means all io.grpc:*, io.netty:*, and `com.google.protobuf:protobuf-java
 # need to be updated with careful consideration.
 
-netty_tcnative_version = "2.0.61.Final"
-netty_version = "4.1.108.Final"
-grpc_version = "1.60.0"
-protobuf_version = "3.24.0"
-pekko_version = "1.0.1"
-pekko_http_version = "1.0.0"
+# Version of GRPC and netty should follow the "known to work version combinations" as
+# described in  https://github.com/grpc/grpc-java/blob/master/SECURITY.md#netty
+netty_tcnative_version = "2.0.66.Final"
+netty_version = "4.1.100.Final"
+grpc_version = "1.65.1"
+protobuf_version = "3.24.4"
```

Comment:
I have troubles to compile 3.25.x so I keep 3.24.x. I am not sure spending time to make 3.5 work on main-2.x is worthwhile.
 grpc 1.66.x does not seem to work with proto 3.24.x.